syntax_sugar/infix.py

```diff
@@ -45,19 +47,19 @@ def __iter__(self):
     
     def __next__(self):
         if self.type == 'number':
-            if self.curr <= self.end:
+            if self.curr == self.end + self.step:
```

Comment:
That could be a plan but does not read natural syntactically.
Another plan could be
``` python
list( (1, 4) /to/ 10) = [1, 4, 7, 10]
list( (10, 7) /to/ 1) = [10, 7, 4, 1]
```

Any way, if you can fix the implementation here. We can try to implement the syntax later more easily.